modules/@apostrophecms/ui/ui/apos/components/AposSlat.vue

```diff
@@ -18,6 +18,11 @@
     >
       <div class="apos-slat__main">
         <drag-icon class="apos-slat__control apos-slat__control--drag" :size="13" />
+        <AposContextMenu
```

Comment:
Should this not be conditional on something? Will every use of the slats involve relationships? This does take this component from something very general to being specifically tied to relationship inputs.

Reply:
Yes, you are totally right. 🤦